build-utils/optional-locale-chunk-plugin.js

```diff
@@ -10,18 +10,24 @@
  * This plugin removes the locale chunks from the app entrypoint's immediate
  * chunk dependants list, ensuring that the compiled entrypoint will execute
  * *without* all locale chunks loaded.
+ *
+ * Note that the locale actually gets initially loaded in the server template and
+ * the frontend code (in `app/translations`) may be redundant.
  */
 const PLUGIN_NAME = 'OptionalLocaleChunkPlugin';
 
-const clearLocaleChunks = chunks =>
-  chunks
+const clearLocaleChunks = chunks => {
+  Array.from(chunks)
```

Comment:
`chunks` is no longer an array in 5.

Reply:
How many tacos do I have to pay you to convert our webpack to typescript. It's probably about time

Reply:
😬